tests/relay_integration/test_metrics_extraction.py

```diff
@@ -106,3 +106,61 @@ def test_all_transaction_metrics_emitted(self):
             # Make sure that all the standard strings are part of the list of common strings:
             non_common_strings = strings_emitted - SHARED_STRINGS.keys()
             assert non_common_strings == known_non_common_strings
+
+    def test_histogram_outliers(self):
+        with Feature(
+            {
+                "organizations:transaction-metrics-extraction": True,
+            }
+        ):
+            event_data = {
+                "type": "transaction",
+                "transaction": "foo",
+                "transaction_info": {"source": "url"},  # 'transaction' tag not extracted
+                "timestamp": iso_format(before_now(seconds=1)),
+                "start_timestamp": iso_format(before_now(seconds=2)),
+                "platform": "javascript",
+                "contexts": {
+                    "trace": {
+                        "op": "pageload",
+                        "trace_id": 32 * "b",
+                        "span_id": 16 * "c",
+                        "type": "trace",
+                    }
+                },
+                "user": {"id": 123},
+                "measurements": {
+                    "fcp": {"value": 999999999.0},
+                    "lcp": {"value": -9999999999.0},
+                },
+            }
+
+            settings = {
+                "bootstrap.servers": "127.0.0.1:9092",  # TODO: read from django settings here
+                "group.id": "test-consumer-%s" % uuid.uuid4().hex,
+                "enable.auto.commit": True,
+                "auto.offset.reset": "earliest",
+            }
+
+            consumer = kafka.Consumer(settings)
+            consumer.assign([kafka.TopicPartition("ingest-performance-metrics", 0)])
+
+            self.post_and_retrieve_event(event_data)
+
+            histogram_outlier_tags = {}
+            for _ in range(1000):
+                message = consumer.poll(timeout=1.0)
+                if message is None:
+                    break
+                bucket = json.loads(message.value())
+                try:
+                    histogram_outlier_tags[bucket["name"]] = bucket["tags"]["histogram_outlier"]
+                except KeyError:
+                    pass
+
+            consumer.close()
+            assert histogram_outlier_tags == {
```

Comment:
This assertion will fail until https://github.com/getsentry/relay/pull/3615 is merged (requires up to date relay docker image).